pkgs/top-level/all-packages.nix

```diff
@@ -22822,6 +22822,8 @@ with pkgs;
 
   yara = callPackage ../tools/security/yara { };
 
+  yaru-theme = callPackage ../misc/themes/yaru { };
```

Comment:
That line should be relocated in the `### DATA` section.

Could we also have this derivation live under `pkgs/data/themes` (next to icons, fonts and such).
I know that ATM most other themes are under `pkgs/misc/themes` but it really is data in the end.